README.rst

```diff
@@ -31,8 +31,12 @@ This project is led by a team of researchers at the University of Michigan and i
 Prerequisites
 ****************
 
-Install all the prerequisites using `dependencies.sh` script provided in the home location of this project (where this README.rst file is found). Supports CentOS7 and Ubuntu20.
+Install all the prerequisites using `install_script.sh` script provided in the home location of this project (where this README.rst file is found). Supports CentOS7, Ubuntu 20.04 LTS and Ubuntu 22.04 LTS.
 
+.. code-block:: bash
+
+    $ chmod +x install_script.sh
```

Comment:
I don't understand why these are two scripts `install_script.sh` and `dependencies.sh` ? I see that `install_script.sh' calls `dependencies.sh` internally. Can't we merge and give a switch between install and upgrade?

Reply:
The issue was that the conda installation needed to be done without sudo perms, so that part of the initial install needs to be separated from the rest of the dependency installations
The install_script.sh script is supposed to be a wrapper script so that the end user only needs to execute one script from their side
The process of upgrading is the same, the user will just run the dependencies script with the 'u' option